test/testutil/driver.c

```diff
@@ -120,16 +117,7 @@ static void finalize(int success)
     if (success)
         ERR_clear_error();
     else
-        ERR_print_errors_cb(err_cb, NULL);
-}
-
-static void helper_printf_stdout(const char *fmt, ...)
-{
-    va_list ap;
-
-    va_start(ap, fmt);
-    test_vprintf_stdout(fmt, ap);
-    va_end(ap);
+        TEST_openssl_errors();
```

Comment:
I'd like to argue in favour of keeping call to ERR_print_errors_cb() as more readable. In sense that both cases refer to same interface family, `ERR_*`, not mixture.